utils/utils.go

```diff
@@ -150,13 +151,14 @@ func CheckStandardStatus(status int, mesg []byte) error {
 
 		message = ScrapeErrorMessage(message, scrapResponse)
 		result := strings.Split(message, ",")
-		if result != nil && len(result) >= 1 {
-			message = result[0]
+		// concatenate as many errors as received
+		subMessages := make([]string, 0)
+		for i, val := range result {
+			// Separate into fields with func.
+			fields := strings.FieldsFunc(val, f)
+			subMessages = append(subMessages, strings.Join([]string{strconv.Itoa(i), strings.Join(fields[:], " ")}, "# "))
```

Comment:
This line is far too complex, don't abuse strings.Join and use fmt.Sprintf instead

Reply:
Done as well as deeper changes